manifest/services.py

```diff
@@ -203,6 +227,12 @@ def __reset_manifest_step(self, step_name: str):
             self.manifest.steps[step_name] = new_manifest_step
             self.__resetted_manifest_steps.add(step_name)
 
+    def _reset_manifest_document_statuses(self) -> None:
+        """Reset manifest document statuses to defaults."""
+        self.manifest.status = ManifestStatus.FAILED
```

Comment:
We could improve this method by enclosing its body in a condition, where it only runs if **self.__is_manifest_loaded** flag is set to _True_, as we only need to reset the _ManifestDocument_ status when loading a pre-existing _manifest file_